## Research areas
Below are my research interests and some of my selected publications under each category.

---
{{< accordion >}}
{{< accordion_item title="Computational biomarker discovery" open="true" >}}
Developed various computational methods, including machine learning and data integration approaches, for identification of predictive biomarkers.

- A novel method on discovering potential multiplatform biomarkers using a combination of data-driven reference genes with non-parametric Fisherâ€™s exact test. [Read more...](https://doi.org/10.1186/s12859-019-3140-7)
- Prediction of clinical outcomes in oligometastatic castration-resistant prostate cancer (omCRPC) treated with SABR using tumor-derived extracellular vesicles.
[Read more...](https://doi.org/10.1016/j.ijrobp.2022.05.037)
- Identification of potential biomarkers of infection in the CSF using a rat model of <em>S. epidermidis</em> catheter infection and proteomics. 
[Read more...](https://doi.org/10.1128/iai.00311-19)
{{< /accordion_item >}}

{{< accordion_item title="Agent-based modeling in biomedicine" >}}
Developed agent-based models (ABM) to study viral propagation of vector-borne disease
in a population and to model adaptive therapy in cancer. Additionally, developed a new ABM generator platform, called BOAST-ABM, which enables creating ABMs by specifying agents and their behaviors in structured specification documents that are translated into executable code using abstract syntax trees. 
- Agent-based modeling of preferential attraction to infected hosts in vector-borne diseases such as dengue fever on the viral transmission in the population. [Read more...](https://doi.org/10.3389/fpubh.2023.1276029)
- Dissertation titled "Democratizing Design and Implementation of Agent-Based Modeling in Biomedicine". [Abstract](/education/doctoral/abstract.pdf)
{{< /accordion_item >}}

{{< accordion_item title="Development of bioinformatics pipelines and software applications" >}}
Developed bioinformatics software including a Shiny app for flow cytometry QC and automated EV gating, a TCGA-based multi-omics graph database for efficient mining, and FunSet for GO enrichment analysis with 2D semantic-similarity visualization
- A new Shiny web application is developed that performs quality check of flow cytometry files and
automated gating of subpopulations of EVs that are of interest to next generation biomarker
studies. [Read more...](https://www.doi.org/10.5220/0013318900003911)
- A new graph database for integration of omics data and its efficient mining. A working model of this graph database with transcriptomics, genomics, epigenetics and clinical data TCGA is presented. [Read more...](https://doi.org/10.1089/cmb.2020.0231) 
- FunSet is an open-source functional enrichment analysis tool, specifically aimed at Gene Ontology (GO) enrichment analysis and interactive visualization of the enriched GO terms. The tool identifies GO terms that are statistically overrepresented and are displayed in a 2D plot based on semantic similarity. [Read more...](https://doi.org/10.1186/s12859-019-2960-9)

{{< /accordion_item >}}

{{< accordion_item title="Complex network applications in biomedicine" >}}
Performed complex network analysis on coexpression of genes involving modeling of gene interactions and regulatory relationship and on co-occurrence relationships between microorganisms within a community based on their simultaneous presence or absence across samples.
- Co-expression models were generated for wild type and miR-142 overexpression neuronal cells along with integration of miRNA seed sequence mapping information to identify genes greatly affected by this overexpression. [Read more...](https://doi.org/10.1155/2015/921517)
- Introduced a novel computational approach for the identification of co-occurring microbial communities using the abundance and functional roles of species-level microbiome data. The proposed approach is then used to identify signature pathways associated with inflammatory bowel disease (IBD). [Read more...](https://doi.org/10.1186/s12866-024-03633-6)
- This research aims to evaluate co-regulation of clusters in gene coexpression networks via multiple clustering approaches and cross-validation of regulatory elements by motif finding software. [Read more...](https://doi.org/10.1109/ICDMW.2013.125)
{{< /accordion_item >}}

{{< /accordion >}}
